src/main/java/io/jenkins/plugins/pipelinegraphview/PipelineGraphDisplayURLProvider.java

```diff
@@ -24,6 +25,30 @@ public String getName() {
     @Override
     @NonNull
     public String getRunURL(Run<?, ?> run) {
-        return getRoot() + Util.encode(run.getUrl()) + "pipeline-graph";
+        if (run instanceof WorkflowRun) {
+            return DisplayURLProvider.getDefault().getRunURL(run) + "pipeline-graph";
+        }
+        return DisplayURLProvider.getDefault().getRunURL(run);
+    }
+
+    @Override
+    public String getArtifactsURL(Run<?, ?> run) {
+        return DisplayURLProvider.getDefault().getArtifactsURL(run);
+    }
+
+    @Override
+    public String getChangesURL(Run<?, ?> run) {
+        return DisplayURLProvider.getDefault().getChangesURL(run);
+    }
+
+    @Override
+    public String getTestsURL(Run<?, ?> run) {
+        return DisplayURLProvider.getDefault().getTestsURL(run);
+    }
+
+    @Override
+    public String getJobURL(Job<?, ?> job) {
+        // TODO: Could redirect this to MultiPipelineGraphViewAction
```

Comment:
Just FYI, this is possible. IDK if you want to do it though, maybe it is not really intended to be a total replacement for the main job page.

Reply:
i had a draft PR at some point to add more to it but never finished it